src/layers/msgpass.jl

```diff
@@ -34,14 +34,18 @@ update_vertex(m::T; kwargs...) where {T<:MessagePassing} = update(m; kwargs...)
 aggregate_neighbors(m::T, aggr::Symbol; kwargs...) where {T<:MessagePassing} =
     pool(aggr, kwargs[:cluster], kwargs[:M])
 
+aggregate_neighbors(m::T, aggr::Symbol, M, cluster; kwargs...) where {T<:MessagePassing} =
```

Comment:
I think `aggregate_neighbors(m::T, aggr::Symbol; kwargs...)` is general enough to provide the flexibility. We shouldn't bother to provide similar functionality for different arguments.
We can add documentation to guide people how to use it.

Reply:
I just thought the kwargs caused some Zygote issues.

Reply:
It's fine in my case.
```
julia> using Zygote
julia> foo(;x=1, y=2) = x+y
foo (generic function with 1 method)

julia> gradient(y -> foo(;x=1,y=y), 4)
(1,)

julia> gradient(x -> foo(;x=x,y=2), 4)
(1,)
```

Reply:
Yes, it was not causing any problem, but it prevented Zygote from showing the right error message.

Reply:
Fine. I will refactor the architecture recently. This will be fixed.